static/lang/re-en.json

```diff
@@ -480,6 +480,10 @@
                 "Fetch": "Subservience - Fetch",
                 "Kill": "Subservience - Kill"
             },
+            "LocathahHuntPrey": {
+                "Damage": "Locathah hits Designated Prey for the first time in a round",
+                "Checks": "Locathah rolls checks against his Designated Prey"
```

Comment:
These could be done more generically so other things with hunt prey like mechanics could use them.